<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adicionar Recurso</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Acesso ao recurso:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>URL:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Nome:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="tx_url"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="tx_name"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>308</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="bt_ok">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="bt_cancel">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tx_name</tabstop>
  <tabstop>tx_url</tabstop>
  <tabstop>bt_ok</tabstop>
  <tabstop>bt_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
